Makefile-test.mk

```diff
@@ -99,6 +94,14 @@ test-integration: gomod-download envtest ginkgo dep-crds kueuectl ginkgo-top ##
 	$(GINKGO) $(INTEGRATION_FILTERS) $(GINKGO_ARGS) $(GOFLAGS) -procs=$(INTEGRATION_NPROCS) --race --junit-report=junit.xml --json-report=integration.json --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/integration.json > $(ARTIFACTS)/integration-top.yaml
 
+.PHONY: test-integration-required
+test-integration-required: INTEGRATION_FILTERS= --label-filter="!slow && !redundant"
+test-integration-required: test-integration ## Run required integration tests for singlecluster suites.
```

Comment:
This test running in 16m14s.

https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/5068/pull-kueue-test-integration-main/1915280153307516928

Reply:
Without slow and redundant 13m9s.